Examples/Errors/local_in_shared_context.cvf

```diff
@@ -12,11 +12,11 @@ thread int s;  // The thread's current view of serving.
  */
 method lock() {
   {| emp |}
-    <t = s++>; // <-- Thread-local in shared context
+    <| t = s++; |> // <-- Thread-local in shared context
```

Comment:
This, and all of the `.cvf` changes (i.e. existing files), are just swapping out the old syntax for the new one.